spec/features/managing_employees_spec.rb

```diff
@@ -80,6 +122,35 @@
           expect(page).to have_content service_name
         end
 
+        scenario 'it check the inputs' do
+          click_link 'Change'
+          expect(page).to have_content 'Edit an employee'
+          expect(page).to have_selector("input[value=#{surname}]")
```

Comment:
You can make this a bit neater with the [`have_field` matcher](https://www.rubydoc.info/gems/capybara/Capybara/Node/Matchers#has_field%3F-instance_method) (`have_field` is an alias for `has_field?` so you can use them interchangeably):

```
expect(page).to have_field(:employee_surname, with: surname)
```